pkg/analyzer/analyzers/slack/slack.go

```diff
@@ -69,21 +74,38 @@ func getSlackOAuthScopes(cfg *config.Config, key string) (scopes string, userDat
 	return scopes, userData, err
 }
 
-func AnalyzePermissions(cfg *config.Config, key string) {
-	scopes, userData, err := getSlackOAuthScopes(cfg, key)
-	if err != nil {
-		color.Red("[!] Error getting Slack OAuth scopes:", err)
+func AnalyzeAndPrintPermissions(cfg *config.Config, key string) {
+	// ToDo: Add in logging
+	if cfg.LoggingEnabled {
+		color.Red("[x] Logging is not supported for this analyzer.")
 		return
 	}
```

Comment:
I think logging is supported here. If the analyzer calls `analyzers.NewAnalyzeClient(cfg)`, then logging is supported.

Reply:
Remove the logging condition and updated the code.